dired-sidebar.el

```diff
@@ -322,6 +322,11 @@ For more information, look up `delete-other-windows'."
   :type 'boolean
   :group 'dired-sidebar)
 
+(defcustom dired-sidebar-one-instance nil
```

Comment:
This should be dired-sidebar-one-instance-p.

Reply:
Got it.